pkg/network/ebpf/c/ipv6.h

```diff
@@ -25,13 +29,8 @@ __maybe_unused static __always_inline bool is_ipv4_mapped_ipv6(__u64 saddr_h, __
 }
 
 static __always_inline void read_in6_addr(u64 *addr_h, u64 *addr_l, const struct in6_addr *in6) {
-#ifdef COMPILE_PREBUILT
-    bpf_probe_read_kernel_with_telemetry(addr_h, sizeof(u64), (void *)&(in6->in6_u.u6_addr32[0]));
-    bpf_probe_read_kernel_with_telemetry(addr_l, sizeof(u64), (void *)&(in6->in6_u.u6_addr32[2]));
-#else
-    BPF_CORE_READ_INTO(addr_h, in6, in6_u.u6_addr32[0]);
-    BPF_CORE_READ_INTO(addr_l, in6, in6_u.u6_addr32[2]);
-#endif
+    *addr_h = *(u64*)(&in6->in6_u.u6_addr32[0]);
+    *addr_l = *(u64*)(&in6->in6_u.u6_addr32[2]);
```

Comment:
Why is it safe to get rid of the prebuilt version here? Is in6_addr a stable struct ?

Reply:
Actually this doesn't seem to work always, I'll revert this change.